ghost/i18n/locales/pt-BR/ghost.json

```
@@ -7,25 +7,25 @@
     "Confirm your email address": "Confirme seu endereço de e-mail",
     "Confirm your email update for {{siteTitle}}!": "Confirme sua atualização de e-mail no site {{siteTitle}}!",
     "Confirm your subscription to {{siteTitle}}": "Confirme sua assinatura no site {{siteTitle}}",
-    "For your security, the link will expire in 24 hours time.": "Para sua segurança, o link será desativado dentro de 24 horas.",
+    "For your security, the link will expire in 24 hours time.": "Por questões de segurança, o link expirará em 24 horas.",
     "Hey there,": "Olá,",
     "Hey there!": "Olá!",
-    "If you did not make this request, you can safely ignore this email.": "Se você não fez esta solicitação, pode ignorar este e-mail.",
-    "If you did not make this request, you can simply delete this message.": "Se você não fez esta solicitação, pode apagar esta mensagem.",
+    "If you did not make this request, you can safely ignore this email.": "Se você não fez esta solicitação, desconsidere este e-mail.",
+    "If you did not make this request, you can simply delete this message.": "Se você não fez esta solicitação, desconsidere esta mensagem.",
     "Please confirm your email address with this link:": "Por favor, confirme seu endereço de e-mail usando este link:",
     "Secure sign in link for {{siteTitle}}": "Link seguro para acesso ao site {{siteTitle}}",
     "See you soon!": "Até logo!",
     "Sent to {{email}}": "Enviado para {{email}}",
     "Sign in": "Entrar",
     "Sign in to {{siteTitle}}": "Entrar no site {{siteTitle}}",
     "Tap the link below to complete the signup process for {{siteTitle}}, and be automatically signed in:": "Clique no link abaixo para completar o processo de cadastro no site {{siteTitle}} e entrar automaticamente:",
-    "Thank you for signing up to {{siteTitle}}!": "Obrigado por se cadastrar no site {{siteTitle}}!",
-    "Thank you for subscribing to {{siteTitle}}!": "Obrigado por se inscrever no site {{siteTitle}}!",
-    "Thank you for subscribing to {{siteTitle}}.": "Obrigado por se inscrever no site {{siteTitle}}.",
-    "Thank you for subscribing to {{siteTitle}}. Tap the link below to be automatically signed in:": "Obrigado por se inscrever no site {{siteTitle}}. Clique no link abaixo para entrar automaticamente:",
+    "Thank you for signing up to {{siteTitle}}!": "Seu cadastro no site {{siteTitle}} é muito apreciado. Gratidão!",
+    "Thank you for subscribing to {{siteTitle}}!": "Sua inscrição no site {{siteTitle}} e muito apreciada. Gratidão!",
```

```suggestion
    "Thank you for subscribing to {{siteTitle}}!": "Agradecemos pela sua inscrição em {{siteTitle"}}!",
```

---

“Agradecemos” is in the plural, and it’s not always that there’s a team behind a website. It could be just one person.

---

Here I use the plural in the sense of service, for example, when it comes to a newsletter, it can be understood as a service offered by the site.
You can use the entry “agradeço”.

But one option for the entry, instead of agradecemos, is to use the word “obrigado”, because in this entry it can be considered a noun and for this it is already adequate, according to Brazilian grammar rules.

But I also think that just saying "obrigado" or "agradeço" you isn't adequate yet.

I'd use part of your entry, but I don't think the word “apreciada” speaks to the general public because it's too formal. And "gratidão" isn't appropriate either, because in many places in Brazil it already has a religious character.

Your entry on {{siteTitle}} is much appreciated.

```suggestion
    "Thank you for subscribing to {{siteTitle}}!": "Obrigado pela sua inscrição em {{siteTitle}}!",
```

---

@zenditimo  estou pensando em alguma opção que seja mais adequada, com menos formalidade. Estou discutindo um verbete com a minha equipe.

---

@zenditimo yes, I know it's not neutral when we consider Neutral Language.
What I pointed out is that for grammar purposes, “thank you” is a neutral word, and I make that clear when I say that I don't believe thank you is an appropriate term. 
Gratitude is a nice word, I use it myself on a daily basis, but I believe we can think of a way of using it that suits any audience. 
I understand the difficulties of adjusting entries, I have to practice this on a daily basis in my work.

One option would then be “Gratidão pela sua inscrição em {{siteTitle}}”.
That way you'd have the word gratitude, but in a less formal tone. 

I chose “em” because it can be used for a wide range of names. For example:

"Gratidão pela sua inscrição em Publiso"; 
"Gratidão pela sua inscrição em TechHub";
"Gratidão pela sua inscrição em Núcleo Jornalismo";
"Gratidão pela sua inscrição em Escritos que Fugiram da Gaveta".

-------

@zenditimo sim, eu sei que não é neutro quando levamos em consideração a Linguagem Neutra.
O que eu pontuei é que para fins de gramática, "obrigado" é uma palavra neutra, e deixo isso claro quando digo que não acredito que obrigado é um termo adequado. 
Gratidão é uma palavra agradável, eu mesmo uso no dia a dia, mas acredito que podemos pensar numa forma em que a palavra que se adeque a qualquer publico. 
Eu entendo as dificuldades de ajustar os verbetes, tenho que praticar isso no dia a dia no meu trabalho.

Uma opção então seria "Gratidão pela sua inscrição em {{siteTitle}}".
Dessa forma teria a palavra gratidão, mas em tom menos formal. 

A escolha "em" é porque pode servir para ampla gama de nomes. Como por exemplo:

"Gratidão pela sua inscrição em Publiso"; 
"Gratidão pela sua inscrição em TechHub";
"Gratidão pela sua inscrição em Núcleo Jornalismo";
"Gratidão pela sua inscrição em Escritos que Fugiram da Gaveta".